### PR TITLE
Setup Location resolvers to work with NewSpring Specific data

### PR DESCRIPTION
**Pull request by IsaacHardy**
### What does this PR do, or why is it needed?
We need to update resolvers to work with null or undefined data. This PR protects from that.

### How do I test this PR?
Go through the on boarding process and make sure newspring campuses are available.
---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [DEV-XXX](#url)
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android if applicable
- [ ] Set two relevant reviewers

## REVIEW

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by richarddubay** [comment]
This is pulling campuses that should be closed. It's also pulling 'Web' ... I'm pretty sure we shouldn't show that as a campus. 

Screenshot 2019-05-02 16 01 03

**Comment by IsaacHardy** [comment]
@richarddubay This PR's code is only to protect against null and undefined values. I believe the issue with bad data comes from Rock specific data. We should be able to do that from the admin panel.

**Comment by richarddubay** [comment]
Okay, that's cool. I was worried that we were pulling campuses that should be closed ... and maybe that would need to be addressed here. But we can look to a later time for that.

**Review by richarddubay — DISMISSED** [review]

**Comment by IsaacHardy** [comment]
@richarddubay I'm updating the createAssetUrl. Had a better idea for it.

File: packages/apollos-church-api/src/data/utils.js [hunk 1]
```diff
@@ -2,9 +2,16 @@ import ApollosConfig from '@apollosproject/config';
...
 
-export const createAssetUrl = (value) =>
-  `${ROCK.IMAGE_URL}/${
-    ROCK_MAPPINGS.ASSET_STORAGE_PROVIDERS[`${value.AssetStorageProviderId}`]
-  }/${value.Key}`;
+export const createAssetUrl = (value) => {
+  if (value.path) {
+    return value.path;
+  }
+  if (value.Key && value.AssetStorageProviderId) {
```
Comment (richarddubay) [inline packages/apollos-church-api/src/data/utils.js:9]: Just covering :allthethings:. Nice.

**Review by richarddubay — APPROVED** [review]

**Pull request merged**